src/App.jsx

```diff
@@ -1,15 +1,122 @@
 import logo from "/logo.png";
 import "./App.css";
+import { useState } from "react";
+import axios from "axios";
+
+const API_KEY = "4383f46f58cb016f45b5ef2b099d8d5a";
 
 function App() {
+  //when user keys in city
+  const [userInputCity, setuserInputCity] = useState("");
+  const [showTemperature, setShowTemperature] = useState("");
+  const [showCityName, setShowCityName] = useState("");
+  const [showForecast, setShowForecast] = useState("");
+
+  //prevent form from refreshing
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    axios
+      //get location cords to map into weather URL
+      .get(
+        `http://api.openweathermap.org/geo/1.0/direct?q=${userInputCity}&limit=5&appid=${API_KEY}`
+      )
+      .then((response) => response.data[0])
+
```

Comment:
Thank you for showing me both portions of base and not, you could use branches to showcase in the future, just let me know in comments, but this is very nicely done!